README.md

```diff
@@ -15,12 +15,17 @@ Things you may want to cover:
 ## Database creation
 
 ## Database initialization
+Once you have the proiject opened in VS Code. Install Postgres. Then Click the PostgresSQL Elephant icon, select the myapp_development dropdown menu, select the public drop down menu, right click users, hover over "Select..." and click "Run Select Top 1000". 
```

Comment:
This does not explain how you init the database, this is more of how you can _see_ data in the database.

To init the database after cloning and installing postgres (which you should reference the wiki link here) you run:
```
rails db:create
rails db:migrate
rails db:seed
```